shared/rspec/support/system.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.configure do |config|
+  config.before(:each, type: :system) do
+    # driven_by :selenium, using: :chrome
+    # driven_by :selenium, using: :firefox
```

Comment:
Let's remove this commented code. Devs can also switch the driver per spec:

```
it 'creates a new record', driver: :firefox do
   ...
end
```

Reply:
@olivierobert Currently RSpec doesn't support to change driver by use `driver: :firefox`

If devs want to change it for specific test, dev should change like this

```
it 'creates a new record' do
  driven_by :selenium, using: :firefox
  ...
end
```

So not sure should I keep the commented code or remove it, if I keep it, devs can be copy and paste faster than write, also if they want to change the driver at global scope, just uncomment, no need to write again. 😄 

How do you think?

Reply:
@anduonghien its because you only register `headless_chrome ` driver in `shared/rspec/support/capybara.rb`.

We need to have multiple defined drivers in case we want to switch

Reply:
I'm not sure I understand what you mean @abhinavmsra 🤔

Reply:
@anduonghien as we discussed, lets add some comments to explain the drivers are available by default and can be changed easily